src-files/def-config.yaml

```diff
@@ -53,5 +53,7 @@ verification:
     - ../../merge_requests
     - ../../merge_requests/*
 
+  # Regular expressions that match external references that have to be ignored.
```

Comment:
I would also mention which regex syntax do we support (looks like it is POSIX in our case)